src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -482,26 +476,21 @@ Follow these instructions to set up mutual TLS authentication for etcd in [OpenS
    kubectl get secret kube-etcd-client-certs -n openshift-monitoring -o yaml > etcd-secret.yaml
    ```
 
-2. Open the secret file and change the keys:
-   * Rename the **certificate authority** to `cacert`.
-   * Rename the **client certificate** to `cert`.
-   * Rename the **client key** to `key`.
-
-3. Optionally, change the secret name and namespace to something meaningful.
+2. Optionally, change the secret name and namespace to something meaningful.
```

Comment:
I opened this https://github.com/newrelic/nri-kubernetes/issues/405, can it be closed?

I was thinking as well to the `oc adm policy add-scc-to-user privileged` instruction. Do you have more context?

Reply:
Not from my side. I simply changed this because newrelic/nri-kubernetes#342 is merged and now the integration reads TLS secrets properly.

But I have not reviewed Openshift installation to feel as comfortable to close your issue.